urbs/input.py

```diff
@@ -155,6 +155,10 @@ def pyomo_model_prep(data, timesteps):
     m.stor_init_bound = m.storage['init']
     m.stor_init_bound = m.stor_init_bound[m.stor_init_bound >= 0]
 
+    # storages with fixed energy-to-power ratio
+    m.stor_en_to_pow = m.storage['en-to-pow']
```

Comment:
I just saw, that the `stor_init_bound` has the same structure, but I wonder why it has the `r` and not just `sto_ep` (see next comment)

Reply:
I renamed `stor_en_to_pow` to `sto_ep_ratio` here. 
Also the column name in the input file from `en-to-pow` to `ep-ratio` . I kept the term "energy" here since E2P (or P2E) ratio is a commonly used technical parameter in literature for storage systems, so I think it would be good for familiarity by the user side.